common/src/main/java/tachyon/network/protocol/databuffer/DataNettyBuffer.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.network.protocol.databuffer;
+
+import java.nio.ByteBuffer;
+
+import io.netty.buffer.ByteBuf;
+
+/**
+ * A DataBuffer with the underlying data being a {@link ByteBuf}.
+ */
+public class DataNettyBuffer extends DataBuffer {
+  private final ByteBuffer mBuffer;
+  private final long mLength;
+  private final ByteBuf mNettyBuf;
+
+  /**
+  * Constructor for creating a DataNettyBuffer, by passing a Netty ByteBuf.
```

Comment:
Here the javadoc seems not formatted (wrapped) correctly. Could you run format in your IDE to double check?